probe_scraper/scrapers/git_scraper.py

```diff
@@ -399,6 +399,17 @@ def scrape(
     upload_repos = []
 
     for repo_info in repos:
+        if not (
+            repo_info.metrics_file_paths
+            or repo_info.ping_file_paths
+            or repo_info.tag_file_paths
+        ):
+            print(
+                f"Skipping commits for repository {repo_info.name}"
+                " because it has no metrics/ping/tag files."
+            )
+            continue
```

Comment:
This would currently skip the following 6 repos: `engine-gecko`, `engine-gecko-beta`, `engine-gecko-nightly`, `lockwise-ios`, `mozillavpn-backend-cirrus`, `monitor-cirrus`

Reply:
I guess that should be fine indeed.